docs/features/data-cli.md

```diff
@@ -2,58 +2,204 @@
 title: "data: Command Line Tool"
 ---
 
-`data` is the command-line tool to prepare, push and get data. With `data` you will be able to:
-
-* Push data online
-* Get data from online sources
-* Get information about particular data files and datasets both locally and remotely including those on the DataHub
-* Validate your data to ensure its quality
-
 [[toc]]
 
-# Install
+## Overview
 
-See [Install `data`][install].
+**"Data-cli"** is an important part of the [DataHub](https://datahub.io/docs/about) project. This is a command line tool, that helps you to manipulate your data (as `git` manipulates the code).
```

Comment:
I've missed it in the previous review but I think you don't need a comma before `that` (I know in Russian you'd need it 😄 )

Reply:
In english you can skip comma here :)
I think English have twice less commas, than Russian, at least I have such a feeling